components/core/UserNav/UserNav.tsx

```diff
@@ -14,7 +14,7 @@ interface Props {
   className?: string
 }
 
-const countItem = (count: number, item: any) => count + item.quantity
+const countItem = (count: number, item: any) => count + (item.quantity > 0 ? 1 : 0)
```

Comment:
This ternary could instead just be: 
`const countItem = (count: number, item: any) => count + (item.quantity > 0)`

Reply:
Hey, the above fix by you sounds just as good if we were using JS since it is a TypeScript project I wanted to make sure that I ensured the type-safety Using your suggested code leads to an error occurring that states: 'Operator + cannot be applied to types 'number' and 'boolean' as shown in the image below

![Screen Shot 2020-11-02 at 4 36 58 AM](https://user-images.githubusercontent.com/29141140/97818005-dacdfd80-1cc5-11eb-816d-06b446c9fd81.png)

Since the change is minor I do not want to go around messing with the Ts settings just so I don't get this error. I am fairly new to React (Learnt React 4 months ago) so I may be in the wrong too.

Reply:
Oooo, right good call 👍. I suppose this is more readable anyway.